code/obj/item/organs/appendix.dm

```diff
@@ -1,40 +1,12 @@
 /obj/item/organ/appendix
 	name = "appendix"
 	organ_name = "appendix"
+	organ_holder_name = "appendix"
+	organ_holder_location = "chest"
+	organ_holder_required_op_stage = 3.0
 	icon_state = "appendix"
 	failure_disease = /datum/ailment/disease/appendicitis
 
-	attack(var/mob/living/carbon/M as mob, var/mob/user as mob)
```

Comment:
Removing all these attack procs that are basically duplicates are good. Not sure why I didn't think of doing that.